server/bin/pbench-index.py

```diff
@@ -61,9 +61,7 @@ def main(options, name):
             7 - Bad metadata.log file encountered
             8 - Error reading a mapping file for Elasticsearch templates
             9 - Error creating one of the Elasticsearch templates
-           10 - Bad hostname in a sosreport
-           11 - Failure unpacking the tar ball
-           12 - generic error, needs to be investigated and can be retried
+            12 - generic error, needs to be investigated and can be retried
```

Comment:
This comment should presumably be removed from here:  the definitions of the error codes are no longer here, so the detailed documentation of them should not be here, either -- it should be with the definitions.  (A reference to where those definitions can be found might be good, but there will be nothing which causes this comment to be updated when the actual definitions are changed, so having this here is potentially worse than useless.)

Reply:
done